charms/istio-pilot/src/charm.py

```diff
@@ -699,7 +698,7 @@ def _patch_istio_validating_webhook(self):
             "Attempting to patch istiod-default-validator webhook to ensure it points to"
             " correct namespace."
         )
-        lightkube_client = Client()
+        lightkube_client = self._lightkube_client
```

Comment:
There shouldn't be here, no.  I can revert this if preferred.The field_manager should only matter if we're `apply`ing.

I added this just for consistency with other cases in the charm.  That way if someone later adds more configuration to self._lightkube_client, there aren't some places that use it and some places that don't.  But reverting this will not change the function here so either is fine with me.